src/appPages/members.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { DataTable } from 'primereact/datatable';
+import { InputText } from 'primereact/inputtext';
+import { Column } from 'primereact/column';
+import { Button } from 'primereact/button';
+import {useData} from '../context/pageContent'
+import { Toast } from 'primereact/toast';
+import { Dialog } from 'primereact/dialog';
+export default function Members() {
+    const [globalFilterValue1, setGlobalFilterValue1] = useState('');
+    const [filters1, setFilters1] = useState(null);
+    const { members, addMember } = useData();
+    const [displayBasic, setDisplayBasic] = useState(false);
+    const [name, setName] = useState('');
+    const [contact, setContact] = useState('');
+    const [location, setLocation] = useState('');
+
+    const memberData ={
+        name,
+        contact,
+        location
+    }
+
+
+    const initFilters1 = () => {
+        setFilters1({
+            // global: { value: null, matchMode: FilterMatchMode.CONTAINS },
+            // name: { operator: FilterOperator.AND, constraints: [{ value: null, matchMode: FilterMatchMode.STARTS_WITH }] },
+            // 'country.name': { operator: FilterOperator.AND, constraints: [{ value: null, matchMode: FilterMatchMode.STARTS_WITH }] },
+            // representative: { value: null, matchMode: FilterMatchMode.IN },
+            // date: { operator: FilterOperator.AND, constraints: [{ value: null, matchMode: FilterMatchMode.DATE_IS }] },
+            // balance: { operator: FilterOperator.AND, constraints: [{ value: null, matchMode: FilterMatchMode.EQUALS }] },
+            // status: { operator: FilterOperator.OR, constraints: [{ value: null, matchMode: FilterMatchMode.EQUALS }] },
+            // activity: { value: null, matchMode: FilterMatchMode.BETWEEN },
+            // verified: { value: null, matchMode: FilterMatchMode.EQUALS }
+        });
```

Comment:
Please remove this commented-out code. It is a bad practice to push it to production. If you still want it, keep it somewhere on your machine

Reply:
Alright